docs/reference/search/search-your-data/search-application-api.asciidoc

```diff
@@ -172,9 +345,9 @@ PUT _application/search_application/my_search_application
   }
 }
 ----
-// TEST[skip:TODO]
```

Comment:
Need to add `// TEST[continued]` here?

Reply:
Nope, we don't need to continue, we only need that if we need the setup from an already created search application. The PUT command doesn't need it.